skunkworks_crow/src/main/java/org/odk/share/bluetooth/BluetoothReceiver.java

```diff
@@ -49,47 +36,18 @@ public void onReceive(Context context, Intent intent) {
         Timber.d(TAG, action);
         BluetoothDevice dev = intent.getParcelableExtra(BluetoothDevice.EXTRA_DEVICE);
         if (dev != null) {
-            Timber.d(TAG + "BluetoothDevice: " + dev.getName() + ", " + dev.getAddress());
+            Timber.d("%s BluetoothDevice: %s, Address: %s", TAG, dev.getName(), dev.getAddress());
```

Comment:
TAG is redundant with Timber as it is already included. So, please remove it

Reply:
fixed.